GitVersionExe/Program.cs

```diff
@@ -124,9 +125,21 @@ static int Run()
                             {
                                 throw new WarningException(string.Format("'{0}' variable does not exist", arguments.ShowVariable));
                             }
-                            Console.WriteLine(part);
+                            jsonOutput =part;
                             break;
                     }
+                    if (string.IsNullOrWhiteSpace(arguments.JsonOutputFile))
+                    {
+                        Console.WriteLine(jsonOutput);
+                    }
+                    else
+                    {
+                        var outfile = new FileInfo(arguments.JsonOutputFile);
```

Comment:
This should be able to just specify a filename and it writes it to the working directory.